doc/release-notes-pr13168.md

```diff
@@ -0,0 +1,6 @@
+Thread names in logs
+--------------------
+
+On platforms supporting `thread_local`, log lines are now prefixed by default
+with the name of the thread that caused the log. To disable this behavior,
+specify `-logthreadnames=0`.
```

Comment:
Why does the OP say it is off by default, but the code and release notes otherwise?

```
 static const bool DEFAULT_LOGTHREADNAMES = true;

Reply:
Wowow, something must've gotten screwed up in a rebase. Fixing - thanks for the catch.